assignment/scripts/loops-practice.js

```diff
@@ -11,27 +11,57 @@ for (let i=0; i<4; i++) {
 }
 
 // 1.a. TODO: Write a for loop to console.log the numbers from 0 to 5 
-//   - Which part of the example loop do you need to change to do this?
+//   - Which part of the example loop do you need to change to do this? 
+// I need to change the middle part. Change number for to number 6.
 console.log('count from 0 to 5');
 
+for (let i=0; i<6; i++) {  
+  console.log(i);
+}
+
+
 // 1.b. TODO: Write a for loop to console.log the numbers from 3 to 5
-//   - Which part of the example loop do you need to change to do this?
+//   - Which part of the example loop do you need to change to do this? 
+// I need to change i=0 to i=3 so it will start counting from 3.
+
 console.log('count from 3 to 5');
 
+for (let i=3; i<6; i++) {  
+  console.log(i);
+}
+
+
 // 1.c. TODO: Write a for loop to console.log EVEN numbers from 2 to 10
-//   - Which part of the example loop do you need to change to do this?
+//   - Which part of the example loop do you need to change to do this? 
+// I added a variable called number with a value of . Then I added a for loop and an if. iside the if i used a Remainder `%` to just get the even numbers.
+
 console.log('count even numbers from 2 to 10 (2, 4, 6, 8, 10):');
 
+for (let i=1; i<=10; i++) {   
+if (i%2==0)   
+{  
+console.log(i);  
+}  
+}  
+
+
+
 // 1.d. (STRETCH) TODO: Write a for loop to do a countdown from 5 to 0
 console.log('STRETCH: countdown from 5 to 0');
 
+for(i = 5; i >= 0; i--) {
```

Comment:
Oh no! Don't forget the `let`! If you omit it, JavaScript will try to make it work, but there will be weird side-effects which could cause hard-to-debug errors elsewhere in your code.